devicon.json

```
@@ -6056,6 +6056,30 @@
         "color": "#DF234F",
         "aliases": []
     },
+    {
+        "name": "netlify",
+        "altnames": [],
+        "tags": [
+            "cloud hosting",
+            "serverless",
+            "dynamic websites",
+            "web applications"
```

They have a [GitHub organization](https://github.com/netlify) with a few open-source projects.
```suggestion
            "web applications",
            "open-source"
```